Docker-Swarm-deployment/single-node/gis/example-secrets/secrets/config.json

```diff
@@ -14,6 +14,9 @@
     {
       "id": "iudx.gis.server.apiserver.ApiServerVerticle",
       "ssl": false,
+      "httpPort": 8080,
+      "keystore": "",
```

Comment:
keystore, keystore Password not required . Its used for tls in vertx but we use nginx and hence would not be needing it. Can be removed .